ucx.spec.in

```diff
@@ -156,6 +156,19 @@ rm -f %{buildroot}%{_libdir}/ucx/lib*.a
 
 %postun -p /sbin/ldconfig
 
+%package static
+Requires: %{name}%{?_isa} = %{version}-%{release}
+Summary: Static libraries required for developing with UCX
+Group: Development/Libraries
+
+%description static
+Provides static libraries required for developing with UCX.
+
+%files static
+%{_libdir}/lib*.a
+%{_libdir}/ucx/lib*.a
+%{_libdir}/pkgconfig/ucx-[!u][!c]*.pc
```

Comment:
can we list specific names?

Reply:
> can we list specific names?

technically yes, but it will require multiple if/endif for every transport similar it is used for CMA module below
do you think it should be changed?

Reply:
I think yes, because such pattern can produce unexpected results

Reply:
> I think yes, because such pattern can produce unexpected results

ok, done